### DISCUSSION TITLE
Build Guide questions

### DISCUSSION THREAD

**Discussion by ducky92** [sanni/cartreader#402] [Need help]
I have a few questions around this build process that I would like clarification on. 

1: the "What to order" page has 
>1x40 20mm long https://www.aliexpress.com/item/32911455899.html 

but the "Build Guide" says to use 25mm male header pins
>Solder the 1K resistor, the 1x8 female pin header and the 1x38 and 1x36 25mm male pin headers to the backside...

which size is actually needed?

2: the "What to order" page has 
>N64 controller extension https://www.aliexpress.com/item/32828382501.html or connector https://www.aliexpress.com/item/32844070827.html

but the build guide doesn't show it anywhere and I am not seeing space for it on the board. 
3: the build guide has us solder a jumper on `J1` but I cannot find this on the board
>On the back you need to close the solder jumper marked in red.

4: more of a comment and a word of caution, the link you have for [the n64 slot is no longer available](https://www.aliexpress.com/item/2251832684708596.html?gatewayAdapt=4itemAdapt) and [the alternative from the same seller](https://www.aliexpress.com/item/2251832643592336.html?spm=a2g0o.order_detail.0.0.252ff19cBJkiQz) has the wrong pin pitch

**Comment by sanni** [comment]
Thanks for the feedback, I have updated the build guide accordingly. 👍
